src/types.rs

```diff
@@ -475,6 +475,13 @@ mod type_definition_tests {
         IslType::anonymous([IslConstraint::element(IslTypeRef::named("int"))]),
         TypeDefinition::anonymous([Constraint::element(0), Constraint::type_constraint(25)])
     ),
+    case::annotations_constraint(
+        /* For a schema with annotations constraint as below:
+            { annotations: closed::[red, blue, green] }
+        */
+        IslType::anonymous([IslConstraint::annotations(vec![text_token("closed")], vec![text_token("red").into(), text_token("blue").into(), text_token("green").into()])]),
```

Comment:
Sure. I can change the methods to accept `Vec<&str>` instead.

Reply:
To be clear, I'm not necessarily asking you to change the signature. I'm asking if is it possible to do something like this where we are using the ISL model instead of an Ion text token that is then converted using `into()`.

```rust
IslType::anonymous([
    IslConstraint::annotations(
        vec![AnnotationModifier::Closed], 
        vec![
            Annotation::new("red", false),
            Annotation::new("green", false),
            Annotation::new("blue", false)
        ]
    )
]),
```

Reply:
Yes, I meant to say that it does make the API to create annotation look easier for user. Although I might just change the boolean to be `enum` instead as It doesn't make it clear from the boolean value how it is actually used.